provision/acc_provision/acc_provision.py

```diff
@@ -1098,11 +1304,19 @@ def Raise(exception):
             (get(("aci_config", "vmm_domain", "nested_inside", "name")),
              required)
 
-    if get(("aci_config", "vmm_domain", "nested_inside", "duplicate_file_router_default_svc")):
+    if not is_chained_mode(config) and get(("aci_config", "vmm_domain", "nested_inside", "duplicate_file_router_default_svc")):
         checks["aci_config/vmm_domain/nested_inside/installer_provisioned_lb_ip"] = \
             (get(("aci_config", "vmm_domain", "nested_inside", "installer_provisioned_lb_ip")),
              required)
 
+    if is_chained_mode(config):
+        if get(("aci_config", "vmm_domain", "mcast_range", "start")):
+            warn(("mcast_range option is not used in %s flavor with chained mode" % config["flavor"]))
+        if get(("aci_config", "vmm_domain", "encap_type")):
+            warn(("encap_type option is not used in %s flavor with chained mode" % config["flavor"]))
+        if get(("aci_config", "vmm_domain", "nested_inside", "installer_provisioned_lb_ip")):
+            warn(("nested_inside option is not used in %s flavor with chained mode" % config["flavor"]))
+
```

Comment:
These checks have to be moved to above where deep_merge of default_config happens. Otherwise we will always print this.

Reply:
Also vlan_range checks at line 1291 should be skipped for chained_mode

Reply:
resolved